library/core/src/num/f128.rs

```diff
@@ -68,6 +68,11 @@ pub mod consts {
     pub const FRAC_1_SQRT_PI: f128 =
         0.564189583547756286948079451560772585844050629328998856844086_f128;
 
+    /// 1/sqrt(2π)
+    #[unstable(feature = "f128", issue = "116909")]
```

Comment:
For this and f16, please also add a comment to note there is a second feature gate:

```rust
#[unstable(feature = "f128", issue = "116909")]
// #[unstable(feature = "more_float_constants", issue = "103883")]
pub const FRAC_1_SQRT_2PI: /* ... */
```